.github/workflows/apply-labels.yml

```diff
@@ -14,7 +14,7 @@ name: 🏷️ Add labels
 
 jobs:
   label:
-    uses: wayofdev/gh-actions/.github/workflows/apply-labels.yml@master
+    uses: wayofdev/gh-actions/.github/workflows/apply-labels.yml@v3.0.0
```

Comment:
Why not
![image](https://github.com/cycle/database/assets/4152481/6aca9e7c-55d9-47d0-a935-c56295eb6b4a)
directly?